apps/testnorge-skd/skd-core/src/main/java/no/nav/registre/skd/provider/rs/SyntetiseringController.java

```diff
@@ -35,7 +35,7 @@
     @ApiOperation(value = "Her bestilles genererering av syntetiske meldinger for nye og eksisterende identer. "
             + "Disse meldingene lagres i angitt gruppe i TPSF. ", notes = "Eksisterende identer hentes fra avspillergruppen og status quo på disse hentes fra TPS i angitt miljø. " +
             "\n\nSpesialbehandlinger: \n\n" +
-            "\tVed bestilling av endringskode 0211, genereres tilhørende endringsmelding 3510.\n\n" +
+            "\tVed bestilling av endringskode 0211, genereres tilhørende endringsmelding 3510. Endringskode 0211 blir nå byttet ut med 5212 i stedet.\n\n" +
```

Comment:
En forskjell på 0211 og 5212 er at eksisterende barn må umiddelbart etterfølges i trans 2 (barnetrans) record. Ved 0211 har man ikke denne begrensning. Forøvrig, fødselsmelding er som før.